README.md

```diff
@@ -53,6 +53,7 @@ To install hivemind from source, simply run the following:
 ```
 git clone https://github.com/learning-at-home/hivemind.git
 cd hivemind
+pip install "grpcio-tools>=1.33.2" # for protobuf compilation
```

Comment:
Perhaps we should just include it to requirements-dev.txt?

Reply:
It is, check the diff and PR description. The rationale is that you don't need to install all the development deps just to make hivemind work

Reply:
Okay, have it your way :)

Reply:
This line is not okay for me, such things should be moved to requirements (that's exactly what requirements are for).

Reply:
Again, it is present in the development requirements file. The issue is that we do not need this package at runtime when we are installing hivemind normally (e.g. from pip), so a regular user does not need it